File: src/uk/dwelsh/kaffee/api/payload/CoffeePayload.java
package uk.dwelsh.kaffee.api.payload;

import uk.dwelsh.kaffee.models.Coffee;

public class CoffeePayload {
    private Coffee type;

    public CoffeePayload(Coffee type) {
        this.type = type;
    }

    @Override
    public String toString() {
        return "CoffeePayload{" +
                "type=" + type +
                '}';
    }
}

File: src/main/kotlin/uk/dwelsh/kaffee/api/payload/CoffeePayload.kt
package uk.dwelsh.kaffee.api.payload

import uk.dwelsh.kaffee.models.Coffee

class CoffeePayload(private val type: Coffee) {
    override fun toString(): String {
        return "CoffeePayload{" +
                "type=" + type +
                '}'
    }

}